sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapi/AADResourceServerConfiguration.java

```diff
@@ -82,17 +83,12 @@ public JwtDecoder jwtDecoder() {
      * bean to override it.
      */
     @Configuration
-    @ConditionalOnMissingBean(WebSecurityConfigurerAdapter.class)
     @EnableWebSecurity
-    public static class DefaultAzureOAuth2ResourceServerWebSecurityConfigurerAdapter extends
-        WebSecurityConfigurerAdapter {
-
+    @ConditionalOnMissingBean(WebSecurityConfigurerAdapter.class)
```

Comment:
We need more condition, webapp and webapi use same condition will cause problem.

Reply:
But the aad web app configuration will not be activated when `BearerTokenAuthenticationToken` is on classpath.

Reply:
It make sense.